test/cookbooks/osl-rt-test/files/default/rt/rt.conf

```
@@ -0,0 +1,17 @@
+RewriteEngine On
+RewriteRule ^/([0-9]+)$ https://request.osuosl.intnet/Ticket/Display.html?id=$1 [QSA,L]
```

I prefer to use this domain for testing.
```suggestion
RewriteRule ^/([0-9]+)$ https://request.example.org/Ticket/Display.html?id=$1 [QSA,L]
```